internal/painter/image.go

```diff
@@ -201,7 +201,8 @@ func isFileSVG(path string) bool {
 	return strings.ToLower(filepath.Ext(path)) == ".svg"
 }
 
-func isResourceSVG(res fyne.Resource) bool {
+// IsResourceSVG checks if given fyne Resource is svg or not
```

Comment:
```suggestion
// IsResourceSVG checks if the resource is an SVG or not.
```

Reply:
@Jacalz sorry for the delay done changes as suggested.

Reply:
No need to apologise ;)